### DISCUSSION TITLE
Customize GitHub links text

### DISCUSSION THREAD

**Discussion by sverchkov** [quarto-dev/quarto-cli#9554] [Q&A] [answered]
### Description

Is there a way to customize the text that shows up for github links (for example if I want the edit link to say "Improve this page" instead of "Edit this page")?

AFAIK there's no way to do it now so it's a feature request.

**Comment by mcanouil** [comment, marked as answer]
Use the lang/language options, see <https://quarto.org/docs/authoring/language.html>.
